pkg/volume/secret/secret.go

```diff
@@ -45,8 +45,10 @@ type secretPlugin struct {
 
 var _ volume.VolumePlugin = &secretPlugin{}
 
-var wrappedVolumeSpec = volume.Spec{
-	Volume: &api.Volume{VolumeSource: api.VolumeSource{EmptyDir: &api.EmptyDirVolumeSource{Medium: api.StorageMediumMemory}}},
+func wrappedVolumeSpec() volume.Spec {
```

Comment:
Well I could do this by adding say `WrappedVolumeSpec()` to pkg/volume/util.go, but that would be a kind of `volume` -> `volume/empty_dir` quasi-dependency which is probably not very pretty.